bessctl/module_tests/ddp-rss-rx.bess

```diff
@@ -1,3 +1,6 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2021 Intel Corporation
```

Comment:
There is 
- `Copyright 2021 Intel Corporation`
- `Copyright (c) 2021 Intel Corporation`
- `Copyright (C) 2021 Intel Corporation`

Can we slim this down to one version, preferably the first?